packages/diet/diet.0.2/opam

```diff
@@ -0,0 +1,33 @@
+opam-version: "2.0"
+maintainer: "dave@recoil.org"
+authors: ["David Scott"]
+license: "ISC"
+homepage: "https://github.com/djs55/ocaml-diet"
+dev-repo: "git+https://github.com/djs55/ocaml-diet.git"
+bug-reports: "https://github.com/djs55/ocaml-diet/issues"
+doc: "https://djs55.github.io/ocaml-diet"
+
+build: [
+  ["dune" "subst" ] {pinned}
+  ["dune" "build" "-p" name "-j" jobs]
+  [make "test"] {with-test}
+]
+depends: [
+  "ocaml" {>= "4.03.0"}
+  "result"
+  "sexplib"
+  "dune" {build}
+  "ppx_tools" {build}
+  "ppx_sexp_conv" {build}
```

Comment:
Hm, good question. I did a bit of digging and found this comment: https://github.com/ocaml/opam-repository/issues/11852#issuecomment-387732193 -- so I guess since this is a PPX rewriter it shouldn't have a `{build}` annotation? I'll remove it.